actions/dataset.go

```diff
@@ -185,8 +183,8 @@ func UpdateDataset(node *p2p.QriNode, ref *repo.DatasetRef, secrets map[string]s
 
 func localUpdate(node *p2p.QriNode, ref *repo.DatasetRef, secrets map[string]string, scriptOut io.Writer, dryRun, pin bool) (res repo.DatasetRef, body cafs.File, err error) {
```

Comment:
While you're here, could you add a comment to function explaining what it does.

Reply:
hey good looking out cause I actually found a bug while reviewing this. making a node in the comment and filing an issue.